include/lomse_logger.h

```diff
@@ -82,13 +80,24 @@ enum {
 class Logger
 {
 private:
+    std::ostream* m_logStream;
+    std::ostream* m_customForensicLogStream;
+    std::ofstream m_forensicLogStream;
     int m_mode;
     uint_least32_t m_areas;
+    bool m_initialized = false;
 
 public:
     Logger(int mode=k_normal_mode);
     ~Logger();
 
+    void init(std::ostream* logStream = nullptr, std::ostream* forensicLogStream = nullptr);
```

Comment:
In the proposed approach, if the applications would like to customize the logging system, it has to do it before creating the LomseDooorway object.

The drawback I see is that in my application and in the tutorials the LomseDoorway object is a member variable of the application class, and thus it is automatically created when the application class is created. In this scenario, initializing the logging system before creating the LomseDoorway object is not a trivial operation that requires changing some code. So, to simplify initialization and for backwards compatibility, the solution could be to change the constructor for the LomseDoorway, and adding to it the same parameters that the `Logger::init()` method:
```
LomseDoorway(std::ostream* logStream=nullptr, std::ostream* forensicLogStream=nullptr)
```
LomseDoorway will then initialize the logging system using those parameters. When nullptr is passed current logging destination will be used.

When `Logger.deinit()` is used the logging system can be re-initialized by using `Logger.init()`.

Reply:
Added these parameters to `LomseDoorway` constructor